jsk_vil_ros/apis/ofa/server.py

```diff
@@ -0,0 +1,278 @@
+# sys
+import os
+# inference
+import torch
+import numpy as np
+import cv2
+from fairseq import utils, tasks
+from fairseq import checkpoint_utils
+from fairseq import options
+from fairseq.dataclass.utils import convert_namespace_to_omegaconf
+from utils.eval_utils import eval_step
+from utils.zero_shot_utils import zero_shot_step
+from tasks.mm_tasks.caption import CaptionTask
+from tasks.mm_tasks.refcoco import RefcocoTask
+from tasks.mm_tasks.vqa_gen import VqaGenTask
+from models.ofa import OFAModel
```

Comment:
this writing may make people confused when you place `utils.py`, `tasks.py` and `models.py` in the same directory.
please use full path like `from fairseq.utils.eval_utils import eval_step`